### DISCUSSION TITLE
Exception 터트리고, Message 안보이는 문제

### DISCUSSION THREAD

**Discussion by her0807** [woowacourse-teams/2022-ternoko#251] [General]
Controller Advice 에서 exception 잡아서 처리해줘서 메세지 못봤음 !! -> //Controller Advice 를 주석 처리하면 에러가 터져서
메세지가 보임

**Comment by Juhyung990122** [comment]
해당사항은 log 머지되면 주석처리없이 해결됩니다~ 어케나오는지는 디스커션 댓글에 있어요
